### DISCUSSION TITLE
tutorials/intro-typescript/04-building-the-schema

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#163] [course feedback]
# tutorials/intro-typescript/04-building-the-schema

https://www.apollographql.com/tutorials/intro-typescript/04-building-the-schema

**Comment by stevegoossens** [comment]
If the Practice questions are not dynamic/random, then the two on this page are repeats from previous pages/stages of this tutorial.

**Reply by mabuyo** [reply]
Ahh, great catch @stevegoossens! Putting on the list to fix ✍🏽
